README.md

```diff
@@ -17,6 +17,17 @@ Dogecoin is a community-driven cryptocurrency that was inspired by a Shiba Inu m
 
 **Website:** [dogecoin.com](https://dogecoin.com)
 
+## Dogecoin Policy 📜
+
+Current blockchain consensus of the Dogecoin network. Such fees, such speed !
```

Comment:
The fee consensus on the blockchain is unfortunately the output of `estimatesmartfee`, which is currently:

```
$ dogecoin-cli estimatesmartfee 1
{
  "feerate": 6.79343758,
  "blocks": 2
}
```

So these are the developer-recommended minimum fees:

```suggestion
Recommended minimum fee parametrization, set as default on the Dogecoin Core wallet:
```

Reply:
Like bitcoin, fees are not fixed ? What does this 6.7 feerate mean ? Like it donate to [#teamseas](https://dogecoin.com/teamseas/) today, I was able to use a fee of 0.01 doge. Actually, the subtlety is not clear to me.

Reply:
Correct, it's not fixed, that's why you cannot call it the blockchain consensus. You "COULD" call it network consensus but then the median fee is exactly what `estimatesmartfee` gives you, and that's insanely high, and changes every block. Therefore the word "consensus" is not correct.

Reply:
How would you list this kind of information to make it simple ? Would be great to be accessible as much as possible, while being accurate

Reply:
We list the recommendation and that's it. I'm personally quite certain that with current adoption, the lower fee works consistently, so that can be recommended now. We just describe what the wallet implements. I made sure that most of it is [documented inline](https://github.com/dogecoin/dogecoin/blob/1.14.5-dev/src/wallet/wallet.h) so that's what we can summarize?